.vscode/launch.json

```diff
@@ -0,0 +1,138 @@
+{
```

Comment:
Presumably this file should not be included? I assume not since you have specifically ignored `.vscode` which this is in.